<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>455</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>455</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>455</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#7395AE;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
font-size:18px;
color:#5D561;
type-text:bold;
border:3px solid gray</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
qproperty-alignment: AlignCenter;</string>
    </property>
    <property name="text">
     <string>GET RESULT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_paid_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
qproperty-alignment: AlignCenter;</string>
    </property>
    <property name="text">
     <string>GET RESULT</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>731</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>81</y>
      <width>28</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NAME</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_name">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>100</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 3px solid gray;
background-color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_taxes">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 3px solid gray;
background-color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_percentage">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:13px;</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_all">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>380</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
qproperty-alignment: AlignCenter;</string>
    </property>
    <property name="text">
     <string>CLEAR ALL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>400</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>By Matei Emil</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>334</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="paidhour">
       <property name="text">
        <string>PAID/HOUR</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="paid_hour">
       <property name="styleSheet">
        <string notr="true"> border: 3px solid gray;
background-color:white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="get_hours_2">
       <property name="styleSheet">
        <string notr="true"> border: 3px solid gray;
background-color:white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="hours_label">
       <property name="text">
        <string>HOURS</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>341</width>
      <height>119</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="5" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="6" colspan="3">
        <widget class="QLabel" name="endlabel">
         <property name="text">
          <string>END</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="9">
        <widget class="QPushButton" name="get_hours">
         <property name="styleSheet">
          <string notr="true">border: 1px solid gray;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
qproperty-alignment: AlignCenter;</string>
         </property>
         <property name="text">
          <string>GET</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="begin_minutes">
         <property name="styleSheet">
          <string notr="true"> border: 3px solid gray;
background-color:white;
placeholder:&quot;min&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="begin_hour">
         <property name="styleSheet">
          <string notr="true"> border: 3px solid gray;
background-color:white;
placeholder=&quot;hour&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="5" colspan="2">
        <widget class="QLineEdit" name="end_hour">
         <property name="styleSheet">
          <string notr="true"> border: 3px solid gray;
background-color:white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QLineEdit" name="end_minutes">
         <property name="styleSheet">
          <string notr="true"> border: 3px solid gray;
background-color:white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_hours">
         <property name="styleSheet">
          <string notr="true">background-color:white;
 border: 3px solid gray;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLabel" name="beginlabel">
         <property name="text">
          <string>BEGIN</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_days">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>WORKING DAYS</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSpinBox" name="days">
       <property name="styleSheet">
        <string notr="true"> border: 3px solid gray;
background-color:white;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="get_no_days">
       <property name="styleSheet">
        <string notr="true">border: 1px solid gray;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
qproperty-alignment: AlignCenter;</string>
       </property>
       <property name="text">
        <string>GET No. DAYS</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TAXES</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>get_paid_2</zorder>
   <zorder>tableWidget</zorder>
   <zorder>label_2</zorder>
   <zorder>label_name</zorder>
   <zorder>label_taxes</zorder>
   <zorder>label_percentage</zorder>
   <zorder>clear_all</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
